HeRoBackEnd/Controllers/RecruitmentController.cs

```diff
@@ -22,6 +23,15 @@ public RecruitmentController(RecruitmentService service, IMapper map)
             mapper = map;
         }
 
+        private int GetUserId()
```

Comment:
This method could be used in many controllers. You can create BaseController and move it there

Reply:
Ok, I will do that